internal/storage/rocksdb/store.go

```diff
@@ -135,6 +135,18 @@ func (rdb *rocksDB) Put(key []byte, value []byte) error {
 	return err
 }
 
+func (rdb *rocksDB) Delete(key []byte) error {
```

Comment:
In addition to a storage layer test (comment below), please include a test inside `internal/slave/service_test.go` to verify if the CDC for delete operations is applied correctly between RocksDB-RocksDB and RocksDB-Badger.

Reply:
Done.